src/boot/efi/boot.c

```diff
@@ -1019,11 +1031,11 @@ static bool menu_run(
                         break;
 
                 case KEYPRESS(0, 0, 'O'): /* Only uppercase, so that it can't be hit so easily fat-fingered, but still works safely over serial */
-                        RT->ResetSystem(EfiResetShutdown, EFI_SUCCESS, 0, NULL);
+                        shutdown_system();
```

Comment:
maybe cast this call to void, to tell static analyzers that we are ignoring the return value on purpose, not by accident